src/rpc/blockchain.cpp

```diff
@@ -1778,6 +1779,11 @@ void CalculatePercentilesByWeight(CAmount result[NUM_GETBLOCKSTATS_PERCENTILES],
     }
 }
 
+void ScriptPubKeyToUniv(const CScript& scriptPubKey, UniValue& out, bool fIncludeHex)
+{
+    ScriptPubKeyToUniv(scriptPubKey, out, fIncludeHex, IsDeprecatedRPCEnabled("addresses"));
+}
```

Comment:
Can you explain why you are using this wrapper approach for `ScriptPubKeyToUniv` but not for `TxToUniv`. Either use it for both or none.